packages/fauna/fauna/resources/indexes/user_by_provider_account_id.fql

```diff
@@ -0,0 +1,9 @@
+CreateIndex({
+  name: "user_by_provider_account_id",
```

Comment:
It would be better to name this `account_by_provider_and_provider_account_id`

Reply:
Changed, thanks!